server/controllers/flora.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose')
+const Flora = mongoose.model('Flora')
+
+module.exports = {
+  getFlora: res => {
+    return new Promise((resolve, reject) => {
+      Flora.find((err, res) => {
+        if (err) reject(err)
+        else resolve(res)
+      })
+    })
+  },
+
+  addFlora: floraObject => {
+    return new Promise((resolve, reject) => {
+      const flora = new Flora()
+      flora.name = floraObject.name
```

Comment:
use a loop and then assign values using `obj[key] = obj2[key]`. Makes it more scalable

Reply:
Look into `Object.keys(obj)`

Reply:
Surely this adds a layer of validation as it discards any fields other than the ones we want.